### DISCUSSION TITLE
Can you set the label color for a disabled QCheckbox?

### DISCUSSION THREAD

**Discussion by eugenevk** [quasarframework/quasar#12123] [General - Components / Directives / etc] [answered]
I have this custom component, which works fine:
```
<template>
  <div class="q-mt-sm">
    <q-checkbox
      :color="readonly ? 'grey' : 'secondary'"
      :disable="readonly"
      :model-value="modelValue"
      @update:model-value="$emit('update:modelValue', $event)"
    >
      <template v-slot:default>
        <div class="text-primary" style="font-size: 12px; letter-spacing: 0px">
          {{ label }}
        </div>
      </template>
    </q-checkbox>
  </div>
</template>

<script>
export default {
  name: 'InputCheckbox',
  emits: ['update:modelValue'],
  props: ['label', 'modelValue', 'readonly'],
  setup (props) {
    return {}
  }
}
</script>
```
Result when enabled:
![image](https://user-images.githubusercontent.com/19785727/150377860-f70f360f-d4db-445b-b722-c376786ae049.png)

Result when disabled:
![image](https://user-images.githubusercontent.com/19785727/150377927-680f1edf-3e20-4308-b5b0-70e61eda750f.png)

In disabled state it looks like the label color gets a different color (it looks like my project's secondary color). Can I set the color manually in my custom component for disabled state?

**Comment by MilosPaunovic** [comment, marked as answer]
[https://codepen.io/milospaunovic/pen/WNZqGEq](https://codepen.io/milospaunovic/pen/WNZqGEq)

**Reply by eugenevk** [reply]
Thanks @MilosPaunovic , but actually, it was the opacity that was causing it. Solved it by adding this to my app.scss:
```
.q-checkbox.disabled {
    opacity: 1 !important;
}
```
